docker/build_deps.sh

```diff
@@ -28,8 +28,8 @@ fetch_git() {
 }
 
 fetch_deps() {
-    fetch_git bloomberg bde-tools 4.8.0.0
-    fetch_git bloomberg bde 4.8.0.0
+    fetch_git bloomberg bde-tools 4.13.0.0
```

Comment:
Should we also synchronize bde/ntf versions in [build-ubuntu.sh](https://github.com/bloomberg/blazingmq/blob/main/bin/build-ubuntu.sh#L84) and [build-darwin.sh](https://github.com/bloomberg/blazingmq/blob/main/bin/build-darwin.sh#L42)?

Reply:
Updated both scripts